src/api/hooks/useBounties.ts

```diff
@@ -1,83 +1,82 @@
-import {ApiPromise} from '@polkadot/api';
-import {BountyIndex, AccountId} from '@polkadot/types/interfaces';
-import type {PalletBountiesBountyStatus, PalletBountiesBounty} from '@polkadot/types/lookup';
-import type {u32} from '@polkadot/types';
-import {DeriveCollectiveProposal} from '@polkadot/api-derive/types';
+import {gql, useQuery} from '@apollo/client';
+import type {ProxyBounty} from 'src/generated/litentryGraphQLTypes';
 
-import useApiQuery from 'src/api/hooks/useApiQuery';
+export type Bounty = ProxyBounty;
 
-export type StatusName = 'Active' | 'Approved' | 'CuratorProposed' | 'Funded' | 'PendingPayout' | 'Proposed';
-
-export interface BountyStatusInfo {
-  beneficiary: AccountId | undefined;
-  status: StatusName;
-  curator: AccountId | undefined;
-  unlockAt: u32 | undefined;
-  updateDue: u32 | undefined;
-}
-
-export type BountyData = {
-  index: BountyIndex;
-  bounty: PalletBountiesBounty;
-  bountyStatus: BountyStatusInfo;
-  description: string;
-  proposals?: DeriveCollectiveProposal[];
-};
-
-export function useBounties() {
-  return useApiQuery('bounties', async (api: ApiPromise) => {
-    const deriveBounties = await api.derive.bounties.bounties();
-    const bounties = deriveBounties.reduce((_bounties, {bounty, description, index, proposals}) => {
-      const _bounty = {
-        index,
-        bounty,
-        bountyStatus: getBountyStatus(bounty.status),
-        description,
-        proposals,
-      };
-      return {..._bounties, [index.toString()]: _bounty};
-    }, {} as Record<string, BountyData>);
-
-    return bounties;
-  });
-}
-
-const getBountyStatus = (status: PalletBountiesBountyStatus): BountyStatusInfo => {
-  const statusAsString = status.type as StatusName;
-
-  let result: BountyStatusInfo = {
-    beneficiary: undefined,
-    status: statusAsString,
-    curator: undefined,
-    unlockAt: undefined,
-    updateDue: undefined,
-  };
-
-  if (status.isCuratorProposed) {
-    result = {
-      ...result,
-      status: 'CuratorProposed',
-      curator: status.asCuratorProposed.curator,
-    };
+export const ACCOUNT_FIELDS = gql`
```

Comment:
Let's create an issue to organize common queries or fragments so we can easily find them to reuse

Reply:
Done!